src/main/pages/che-7/end-user-guide/proc_mounting-a-secret-as-a-file-into-a-workspace-container.adoc

```diff
@@ -5,6 +5,8 @@
 [id="mounting-a-secret-as-a-file-into-a-workspace-container_{context}"]
 = Mounting a secret as a file into a workspace container
 
+WARNING: We use Kubernetes VolumeMount `subPath` feature to mount files into containers. This is supported and enabled by default since Kubernetes v1.15 and OpenShift 4.
```

Comment:
```suggestion
WARNING: {prod} uses Kubernetes VolumeMount `subPath` feature to mount files into containers. This is supported and enabled by default since Kubernetes v1.15 and OpenShift 4.
```

Reply:
Or, as an alternative, `Kubernetes VolumeMount `subPath` feature is used`